### PR TITLE
Fixes related to verbose logging

### PR DESCRIPTION
**Pull request by DonJayamanne**
Found that though we have some logging the messages were not written to the logs.
Basically I was expecting to see some messages and didn't see them

### CODE REVIEW

File: src/platform/logging/index.ts [hunk 1]
```diff
@@ -302,6 +302,9 @@ export function logTo(logLevel: LogLevel, message: string, ...args: Arguments):
         case LogLevel.Debug:
             traceVerbose(message, ...args);
             break;
+        case LogLevel.Trace:
+            traceVerbose(message, ...args);
```
Comment (DonJayamanne) [inline src/platform/logging/index.ts:306]: This is what was missing, the messages were not being written
I think this needs to be cleaned up, we have too many log levels (will look into this in debt, but for now fixed the logging)

File: src/platform/logging/util.ts [hunk 2]
```diff
@@ -6,6 +6,9 @@
 export type Arguments = any[];
 
 function valueToLogString(value: unknown, kind: string): string {
+    if (Array.isArray(value)) {
+        return value.map((item) => valueToLogString(item, kind)).join(', ');
```
Comment (DonJayamanne) [inline src/platform/logging/util.ts:10]: Improves logging of values if the are an array.

File: src/platform/logging/index.ts [hunk 3]
```diff
@@ -128,7 +128,7 @@ export function logValue<T>(property: keyof T) {
         const params = parameterInfos.get(methodName)!;
         params.push({
             parameterIndex,
-            propertyOfParaemterToLog: property as string
+            propertyOfParameterToLog: property as string
```
Comment (DonJayamanne) [inline src/platform/logging/index.ts:131]: Fixed typo

File: src/platform/common/process/proc.node.ts [hunk 4]
```diff
@@ -195,7 +195,7 @@ export class ProcessService extends EventEmitter implements IProcessService {
     }
 
     @traceDecoratorVerbose('Execing shell command', TraceOptions.BeforeCall | TraceOptions.Arguments)
-    public shellExec(command: string, options: ShellOptions = {}): Promise<ExecutionResult<string>> {
+    public shellExec(command: string, @ignoreLogging() options: ShellOptions = {}): Promise<ExecutionResult<string>> {
```
Comment (DonJayamanne) [inline src/platform/common/process/proc.node.ts:198]: Reduce noise in logs

File: src/kernels/kernelFinder.base.ts [hunk 5]
```diff
@@ -55,8 +62,8 @@ export abstract class BaseKernelFinder implements IKernelFinder {
     public async rankKernels(
         resource: Resource,
         notebookMetadata?: nbformat.INotebookMetadata,
-        preferredInterpreter?: PythonEnvironment,
-        cancelToken?: CancellationToken,
+        @logValue<PythonEnvironment>('uri') preferredInterpreter?: PythonEnvironment,
```
Comment (DonJayamanne) [inline src/kernels/kernelFinder.base.ts:65]: Improve logging, we only want to see the Uri.
Reduces noise.

**Review by rchiodo — APPROVED** [review]

**Pull request merged**